src/components/common/RegPeopleAndRoles.vue

```diff
@@ -18,36 +18,41 @@
       <div class="subhead">Your application must include the following:</div>
       <ul>
         <template v-for="(rule, index) in getPeopleAndRolesResource.rules">
-          <li v-if="rule.id === RuleIds.NUM_COMPLETING_PARTY" :key="index">
-            <v-icon v-if="validNumCompletingParty" color="green darken-2" class="cp-valid">mdi-check</v-icon>
-            <v-icon v-else-if="getShowErrors" color="error" class="cp-invalid">mdi-close</v-icon>
-            <v-icon v-else>mdi-circle-small</v-icon>
-            <span class="rule-item-txt">{{rule.text}}</span>
-          </li>
-          <li v-if="rule.id === RuleIds.NUM_PROPRIETORS" :key="index">
-            <v-icon v-if="validNumProprietors" color="green darken-2" class="proprietor-valid">mdi-check</v-icon>
-            <v-icon v-else-if="getShowErrors" color="error" class="proprietor-invalid">mdi-close</v-icon>
-            <v-icon v-else>mdi-circle-small</v-icon>
-            <span class="rule-item-txt">{{rule.text}}</span>
-          </li>
-          <li v-if="rule.id === RuleIds.NUM_PARTNERS" :key="index">
-            <v-icon v-if="validNumPartners" color="green darken-2" class="partner-valid">mdi-check</v-icon>
-            <v-icon v-else-if="getShowErrors" color="error" class="partner-invalid">mdi-close</v-icon>
-            <v-icon v-else>mdi-circle-small</v-icon>
-            <span class="rule-item-txt">{{rule.text}}</span>
-          </li>
-          <li v-if="rule.id === RuleIds.NUM_APPLICANT_PERSON" :key="index">
-            <v-icon v-if="validApplicantPerson" color="green darken-2" class="dir-valid">mdi-check</v-icon>
-            <v-icon v-else-if="getShowErrors" color="error" class="dir-invalid">mdi-close</v-icon>
-            <v-icon v-else>mdi-circle-small</v-icon>
-            <span class="rule-item-txt">{{rule.text}}</span>
-          </li>
-          <li v-if="rule.id === RuleIds.NUM_APPLICANT_ORG" :key="index">
-            <v-icon v-if="validApplicantOrg" color="green darken-2" class="dir-valid">mdi-check</v-icon>
-            <v-icon v-else-if="getShowErrors" color="error" class="dir-invalid">mdi-close</v-icon>
-            <v-icon v-else>mdi-circle-small</v-icon>
-            <span class="rule-item-txt">{{rule.text}}</span>
-          </li>
+          <RuleListItem class="completing-party-rule pt-2"
+            v-if="rule.id === RuleIds.NUM_COMPLETING_PARTY"
+            :key="index"
+            :valid="validNumCompletingParty"
+            :showErrors="getShowErrors"
+            :text="rule.text"
+          />
+          <RuleListItem class="proprietors-rule pt-2"
```

Comment:
Note that the spacing class is here in the parent, not in the sub-component. This allows the parent to control the spacing as it sees fit without "inheriting" whitespace from the component. Another way to say this is that the spacing in the component itself is "tight" -- which may be useful if the component is reused on a different page with different spacing requirements.